### PR TITLE
[textanalytics] release prep

### PR DESCRIPTION
**Pull request by kristapratico**

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
 - Sentiment Analysis
-- Entity Recognition (Named, Linked, and Personally Identifiable Information (PII) entities)
+- Named Entity Recognition
 - Language Detection
 - Key Phrase Extraction
+- Entity Linking
 - Multiple Analysis
-- Healthcare Entities Analysis
+- Personally Identifiable Information (PII) Detection
+- Text Analytics for Health
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/README.md:12]: Aligning this list with the names used in service documentation

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 2]
```diff
@@ -528,10 +530,10 @@ Note: The Healthcare Entities Analysis service is only available in the Standard
 - Linked Entity Recognition
 - Key Phrase Extraction
 - Sentiment Analysis
-- Custom Entity Recognition (see sample [here][recognize_custom_entities_sample])
-- Custom Single Label Classification (see sample [here][single_label_classify_sample])
-- Custom Multi Label Classification (see sample [here][multi_label_classify_sample])
-- Healthcare Entities Analysis (see sample [here][healthcare_action_sample])
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/README.md:534]: these link to the bespoke methods so removing from the "multiple analysis" section

**Review by mssfang — APPROVED** [review]

**Review by deyaaeldeen — APPROVED** [review]

**Review by heaths — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md [hunk 3]
```diff
@@ -2,13 +2,8 @@
 
 ## 5.2.0 (Unreleased)
```
Comment (mssfang) [inline sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:3]: make sure you have a PR to update the date

Comment (kristapratico) [reply sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:3]: Will do in another PR 👍

**Pull request merged**